instrumentation/aws-sdk/aws-sdk-1.11/javaagent/src/test/resources/logback-test.xml

```diff
@@ -9,10 +9,12 @@
     </layout>
   </appender>
 
-  <root level="DEBUG">
+  <root level="INFO">
     <appender-ref ref="console"/>
   </root>
 
-  <logger name="org.elasticmq" level="debug" />
-
+  <logger name="org.elasticmq" level="INFO" />
```

Comment:
You already have root at Info

Reply:
Yeah, I left those separate ones for easier debugging in case of problems (just flip the level, no need to add another logger and recall what was the correct package name).